### PR TITLE
Add parsing problems

### PR DESCRIPTION
**Pull request by MiriamPragmalingu**
Added some information about the parsing problems to the data set comparisson

### CODE REVIEW

**Comment by MiriamPragmalingu** [comment]
> This is my first time reading this file but there are quite a few things I'd like to change in the data-comparisson.md. Not only regarding the particular changes in this commit;
> 
> I'll make a pull request for you Miriam with my suggestions ;)
> 
> They're mostly suggestion, I accept different takes on them, as long as we change those lines. A lot of these are either spellchecking; punctuation; or the ones I want you to re-read and see if you agree with them -> the sentences that took me re-reading two to three times to get the meaning. I tried to give them a different spin, change the order of the sentence or use some synonyms.
> 
> But now I stepped into same problem in that I've wrote these but don't know if they might sound weird to someone else xD So please check!!

Sorry I think I clicked the re-request button on acident, I can't find you're changes somehow, I only see mine

**Comment by dario-alv** [comment]
Weird 0.o Well, I'll make a new one ;)

**Comment by SamyAteia** [comment]
> > This is my first time reading this file but there are quite a few things I'd like to change in the data-comparisson.md. Not only regarding the particular changes in this commit;
> > I'll make a pull request for you Miriam with my suggestions ;)
> > They're mostly suggestion, I accept different takes on them, as long as we change those lines. A lot of these are either spellchecking; punctuation; or the ones I want you to re-read and see if you agree with them -> the sentences that took me re-reading two to three times to get the meaning. I tried to give them a different spin, change the order of the sentence or use some synonyms.
> > But now I stepped into same problem in that I've wrote these but don't know if they might sound weird to someone else xD So please check!!
> 
> Sorry I think I clicked the re-request button on acident, I can't find you're changes somehow, I only see mine

You can see them if you click on the "view changes" button on the right above darios comment. 
https://github.com/pragmalingu/pragmalingu/pull/20/files/136a6cb1c98ec680950329bbb9edd6e9eadd6e26

**Comment by MiriamPragmalingu** [comment]
> > > This is my first time reading this file but there are quite a few things I'd like to change in the data-comparisson.md. Not only regarding the particular changes in this commit;
> > > I'll make a pull request for you Miriam with my suggestions ;)
> > > They're mostly suggestion, I accept different takes on them, as long as we change those lines. A lot of these are either spellchecking; punctuation; or the ones I want you to re-read and see if you agree with them -> the sentences that took me re-reading two to three times to get the meaning. I tried to give them a different spin, change the order of the sentence or use some synonyms.
> > > But now I stepped into same problem in that I've wrote these but don't know if they might sound weird to someone else xD So please check!!
> > 
> > 
> > Sorry I think I clicked the re-request button on acident, I can't find you're changes somehow, I only see mine
> 
> You can see them if you click on the "view changes" button on the right above darios comment.
> https://github.com/pragmalingu/pragmalingu/pull/20/files/136a6cb1c98ec680950329bbb9edd6e9eadd6e26
> 
> image

I clicked on that but it only shows the changes I did, like adding the parsing problems. Not sure why

**Comment by MiriamPragmalingu** [comment]
> Weird 0.o Well, I'll make a new one ;)

I can see the merge request now, so I'll look at that soon 👍

**Comment by SamyAteia** [comment]
hm ok strange, maybe i'm wrong and the button is for something else, i will try doing a review again. We have to learn this 😃

File: docs/benchmarks/data-comparisson.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ ETC. AUTOMATIC DOCUMENT CLASSIFICATION, CLUSTERS, CLUSTERING, TERM
 CLASSIFICATIONS. #
 ```
 
-**Relevance assessments:**<br />In `lisarj.num` are the relevance assessments represented with IDs. Every query-ID (first column) is followed by document-IDs that are considered as relevant. Depending on the program used to open or parse the file the relevant document-IDs could continue in the next line. Since there is no end marker it would be best to parse the lines depending on the query-ID.
+**Relevance assessments:**<br />In `lisarj.num` are the relevance assessments represented with IDs. Every query-ID (first column) is followed by the number of relevant documents-IDs before the relevant document-IDs start. Depending on the program used to open or parse the file the relevant document-IDs could continue in the next line. Since there is no end marker it would be best to parse the lines depending on the query-ID and number of relevant documents IDs.
```
Comment (SamyAteia) [inline docs/benchmarks/data-comparisson.md:52]: This is just a test comment to start a review, i think the changes look fine

File: docs/benchmarks/data-comparisson.md [hunk 2]
```diff
@@ -64,6 +64,10 @@ CLASSIFICATIONS. #
 
 **Other files:**<br />The `lisa.rel`file is an old version of the relevance assessments which helps to understand new version, but is not considered important to use. 
 
+**Parsing problems**:<br />
+* documents: they are all in seperate files, the newline as online marker to seperate ID, title and text can be a bit tricky
```
Comment (SamyAteia) [inline docs/benchmarks/data-comparisson.md:68]: Actual review: online marker shoul be "only marker"?

File: docs/benchmarks/data-comparisson.md [hunk 3]
```diff
@@ -64,6 +64,10 @@ CLASSIFICATIONS. #
...
+* relevance assessments: not parseable line by line, the number of relevant documents should be used to parse
```
Comment (SamyAteia) [inline docs/benchmarks/data-comparisson.md:69]: Sentence seems cut off, missing some information

**Review by SamyAteia — COMMENTED** [review]
Added a review with some comments and suggestions

**Comment by SamyAteia** [comment]
Ok @dario-alv message and github confused me the button actually only points to your changes @MiriamPragmalingu i thought dario wrote a review + pull requests and that the review would be hidden behind this button but it seems if you write a review it will attached the review comments to the changes files, thats why it points to the changes files but there weren't any comments from dario.

**Comment by dario-alv** [comment]
Yes @SamyAteia; sorry!!! I was not clear! This pull request had nothing in particular I wanted to change, but I read the entire .md file for the first time and, as you pointed out, there were quite a few instances of cut-of english I wanted to add a correction to. So I made another pull request to add to this one before merging. Since I don't know if my corrections improved readability for everyone I asked Miriam her opinion on it first before merging them here ;)

File: docs/benchmarks/data-comparisson.md (hunk 1 above)
Comment (MiriamPragmalingu) [reply docs/benchmarks/data-comparisson.md:52]: This way I can see the review. I think when I change something and commit it should show here and you can resolve it when it's done  :thumbsup:

File: docs/benchmarks/data-comparisson.md (hunk 3 above)
Comment (MiriamPragmalingu) [reply docs/benchmarks/data-comparisson.md:69]: I only added them as bullet points so I chose short sentences but the information is all in this sentence, should I change it to a normal sentence?

File: docs/benchmarks/data-comparisson.md (hunk 2 above)
Comment (MiriamPragmalingu) [reply docs/benchmarks/data-comparisson.md:68]: Oh yeah of course, I will change that

**Pull request merged**